mesheryctl/internal/cli/root/system/context/view.go

```diff
@@ -60,3 +108,19 @@ func getYAML(strct interface{}) string {
 	out, _ := yaml.Marshal(strct)
 	return string(out)
 }
+
+func addTokenLocation(c *config.Context) (*contextWithLocation, bool) {
```

Comment:
```suggestion
func getContextWithTokenLocation(c *config.Context) (*contextWithLocation, bool) {
```
Can we rename it like this?

Reply:
@piyushsingariya  Changed the name as you suggested.